tests/test_litellm.py

```diff
@@ -6,8 +6,8 @@
 
 @pytest.mark.parametrize("model_name,expected_tuple", [
     ("gpt-4o", ("openai", "gpt-4o")),
-    ("claude-3-opus", ("anthropic", "claude-3-opus-20240229")),
-    ("claude-3-5-sonnet", ("anthropic", "claude-3-5-sonnet-20240620")),
+    ("claude-3-opus", ("anthropic", "claude-3-opus-latest")),

```

Comment:
@samuelrince Quick question, isn't it dangerous to specify the `latest`? If it changes, ecologits will still use the number of parameters we encoded. We should maybe set-up a notification (or similar) when the model changes.

Reply:
Thanks for the complete answer. 😃 

RIght! Could be a good improvement for `ecologits-data`, but definitely not a priority.. I'll still open a feature request in the repo.